src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
```

Comment:
info function should refer to THIS robot
```suggestion
      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
```